File: java/pca-model/src/main/java/com/sdl/web/pca/client/contentmodel/pagemodeldata/IPagination.java
package com.sdl.web.pca.client.contentmodel.pagemodeldata;

public interface IPagination {
    int First = 0;
    String After = null;

    public int getFirst();
    public void setFirst(int first);

    public String getAfter();
    public void setAfter(String after);
}

File: net/src/Sdl.Tridion.Api.Client/ContentModel/IPagination.cs
namespace Sdl.Tridion.Api.Client.ContentModel
{
    /// <summary>
    /// Cursor-Based Pagination
    /// </summary>
    public interface IPagination
    {
        /// <summary>
        /// Return the first n number of results
        /// </summary>
        int First { get; set; }

        /// <summary>
        /// Start returning results after the specified cursor. 
        /// <remarks>Place your cursor on edges in your query.</remarks>
        /// </summary>
        string After { get; set; }
    }
}
